src/Xrefcheck/Verify.hs

```diff
@@ -138,61 +142,90 @@ data VerifyError
 instance Given ColorMode => Buildable VerifyError where
   build = \case
     LocalFileDoesNotExist file ->
-      "⛀  File does not exist:\n   " +| file |+ "\n"
+      [int||
+      ⛀  File does not exist:
+         #{file}
+      |]
 
     LocalFileOutsideRepo file ->
-      "⛀  Link targets a local file outside repository:\n   " +| file |+ "\n"
+      [int||
+      ⛀  Link targets a local file outside repository:
+         #{file}
+      |]
 
     AnchorDoesNotExist anchor similar ->
-      "⛀  Anchor '" +| anchor |+ "' is not present" +|
-      anchorHints similar
+      [int||⛀  Anchor '#{anchor}' is not present|] +| anchorHints similar
```

Comment:
`anchorHints` does manual string concatenation + manual newlines + `fmt` >_< I think we can get rid of it like this:

```suggestion
      | null similar ->
          [int||
            ⛀  Anchor '#{anchor}' is not present
          |]
      | otherwise ->
          [int|s|
            ⛀  Anchor '#{anchor}' is not present, did you mean:
            #{indentF 4 $ blockListF similar}
          |]
```

Reply:
Also just noticed we don't have bats tests for this "anchor hints" feature. Can you please add one while we're here?

Reply:
Same thing for "ambiguous anchor", can you please add a test for that too?

Reply:
Both done.

Reply:
Brilliant, thanks! :D